packages/shared/src/experiments.ts

```
@@ -624,14 +625,24 @@ export function isSuspiciousUplift(
   baseline: SnapshotMetric,
   stats: SnapshotMetric,
   metric: { maxPercentChange?: number },
-  metricDefaults: MetricDefaults
+  metricDefaults: MetricDefaults,
+  differenceType: DifferenceType
 ): boolean {
   if (!baseline?.cr || !stats?.cr) return false;
 
   const maxPercentChange =
     metric.maxPercentChange ?? metricDefaults?.maxPercentageChange ?? 0;
 
-  return Math.abs(baseline.cr - stats.cr) / baseline.cr >= maxPercentChange;
+  if (differenceType === "relative") {
+    return (stats.expected ?? 0) >= maxPercentChange;
+  } else if (differenceType === "absolute") {
+    return (Math.abs(stats.expected ?? 0) / baseline.cr) >= maxPercentChange;
```

nit - I would put Math.abs() around `baseline.cr` in the denominator.  If the control mean is negative, and the lift is large negative, we want to throw an error.